composite-network.txt

```diff
@@ -0,0 +1,39 @@
+
+
+        ************        ************        *****************
+        * Endorser *        * Verifier *        * Relying Party *
+        ************        *  Owner   *        *  Owner        *
+              |             ************        *****************
+              |                   |                     |
+              |Endorsements       |Appraisal            |Appraisal
+              |                   |Policy for           |Policy for
+              |                   |Evidence             |Attestation
+              |                   |                     |Result
+              v                   v                     |
+         .-----------------------------.                |
+         |           Verifier          |------------.   |
+         '-----------------------------'            |   |
+          ^               | ^            Attestation|   |
+          |               | |                Results|   |
+          |   Endorsements| |                       |   |
+  Evidence|   & Appraisal | |Attestation            v   v
+  of      |   Policy for  | |Results of      .-----------------.
```

Comment:
Regarding "Attestation Results of Subordinate Devices", the roles have attestation results as going into a Relying Party, not a Verifier as this diagram has, so this is confusing.

Reply:
can we collapse the two lines into "Evidence of Composite Device"? (Leader and Subordinate Units).

Reply:
The leader unit/device verifies the evidence of other subordinate units/devices, puts this verification result into the Evidence of the whole Composite Device and conveys the final Evidence to the Verifier.
So, the two lines can be collapsed into one.